### DISCUSSION TITLE
Cannot use in iOS 15 (Pulse 5.1.0+)

### DISCUSSION THREAD

**Discussion by ios-jongwon** [kean/Pulse#307] [Q&A] [answered]
It was possible to use Pulse 5.0.0 on iOS 15, but starting from version 5.1.0, does it require iOS 16 or later?
<img width="273" alt="스크린샷 2024-11-22 오후 5 11 39" src="https://github.com/user-attachments/assets/5a98a079-d3f2-4ffb-963e-2e161fc37d9e">

**Comment by kean** [comment, marked as answer]
Hey, the console now requires iOS 16 because it uses a bunch of APIs only available on that version.
